assets/data/schema.pbtxt

```diff
@@ -0,0 +1,26 @@
+feature {
+  name: "title"
+  type: BYTES
```

Comment:
Shouldnt the type be `STRING`?
<img width="455" alt="image" src="https://user-images.githubusercontent.com/24796206/171933189-76afbd39-ea37-4744-8554-be1f77bc2bd9.png">

Reply:
The `infer_schema` function has `max_string_domain_size: int = 100`. I don't know enough about the fields of the schema proto to know if it's a good idea to increase this enough for the fields to be classified as strings.